src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseServiceTests.cs

```diff
@@ -48,6 +49,36 @@ public class ReleaseServiceTests
             Id = Guid.NewGuid()
         };
 
+        [Fact]
+        public async Task CreateRelease_ReleaseTypeExperimentalStatistics_ReturnsValidationActionResult()
+        {
+            var releaseCreateRequest = new ReleaseCreateRequest
+            {
+                Type = ReleaseType.ExperimentalStatistics,
+            };
+
+            ReleaseService releaseService = BuildReleaseService(Mock.Of<ContentDbContext>());
+
+            Either<ActionResult, ReleaseViewModel> result = await releaseService.CreateRelease(releaseCreateRequest);
```

Comment:
Hey there, we'd normally shorten this to:

`var result = await releaseService.CreateRelease(releaseCreateRequest);`

Similarly, `var releaseService = BuildReleaseService(Mock.Of<ContentDbContext>());`

Reply:
Done :) 

We have discussed this in the backend style-guide session and have agreed that it is a coding standard we have adopted and will take moving forwards